lib/Agrammon/Web/Routes.pm6

```diff
@@ -5,100 +5,67 @@ use Cro::HTTP::Router;
 use Agrammon::Web::Service;
 use Agrammon::Web::SessionUser;
 
-
 sub routes(Agrammon::Web::Service $ws) is export {
+    subset LoggedIn of Agrammon::Web::SessionUser where .logged-in;
```

Comment:
It's clearer to put this entirely outside of `routes`, at the top level, given it's package-scoped.

Reply:
Moved